conf/application.conf

```diff
@@ -105,7 +106,8 @@ microservice {
     upscan-initiate {
         protocol = http
         host = localhost
-        port = 9570
+        port = 7022
```

Comment:
don't we want it to point to the actual upscan stubs locally?

Reply:
That's code I've accidentally left in when testing. Will revert back.

Reply:
fixed